src/code_e92d0_len_5da0.c

```diff
@@ -134,24 +139,19 @@ INCLUDE_ASM(code_e92d0_len_5da0, si_handle_case_equal_AND);
 INCLUDE_ASM(code_e92d0_len_5da0, si_handle_end_case_group);
 
 s32 si_handle_break_case(script_context* script) {
-    if (script->switchDepth < 0) {
-        while (1) {}; //todo INF_LOOP
-    }
+    ASSERT(script->switchDepth >= 0);
     script->ptrNextLine = si_goto_end_case(script);
     return 2;
 }
 
 s32 si_handle_end_switch(script_context* script) {
     s32 switchDepth = script->switchDepth;
-    
-    if (switchDepth < 0) {
-        inf_loop: goto inf_loop; // todo macro? how to do without label
-    }
 
-    script->switchBlockState[script->switchDepth] = 0;
+    ASSERT(switchDepth >= 0);
+
+    script->switchBlockState[switchDepth] = 0; // here
```

Comment:
`// here` can be removed. thanks for matching this with the macro!

Reply:
Oh, whoops, will remove this.